config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
   root to: 'home#show'
   resources :users, only: [:new, :create, :show, :update]
+  post :sms, to: 'sms/users#update'
```

Comment:
Twilio doesn't need it to be SMS. We could namespace it to `api`, but that feels a little odd to me if we want to then expose other API functionality, but don't want this mixed up in it.

One thing I'm thinking: maybe we make this mirror auth callbacks? I'm thinking something like `/messaging/twilio/users` (which to me feels like `/auth/:provider/callback`). I'm totally open to suggestions, but if you think that sounds better let me know and I'll make the change!